geoext.ux/ux/SimplePrint/lib/GeoExt.ux/SimplePrint.js

```diff
@@ -472,16 +477,12 @@ GeoExt.ux.SimplePrint = Ext.extend(Ext.form.FormPanel, {
                         self.printProvider.download(
                             self.printProvider.getDownloadURL(job)
                         );
-                        self.progressPanel.remove(statusComponent);
-                        statusComponent.destroy();
-                        statusComponent = null;
-                        self.progressPanel.doLayout();
-                        self.progressPanel.setHeight(35 * self.progressPanel.items.length);
-                        self.doLayout();
                         return false;
                     };
-                }
-                else {
+                    self.printProvider.download(
+                        self.printProvider.getDownloadURL(job)
+                    );
```

Comment:
For me it's not coherent, you directly do the download and add an URL ?
If we do a directly download we also should remove the panel from the queue...